python-client/giskard/core/suite.py

```diff
@@ -27,6 +27,8 @@
     TestMessageLevel,
 )
 from giskard.models.base import BaseModel
+from giskard.integrations.wandb.wandb_utils import wandb_run
```

Comment:
Put this one inside `to_wandb` otherwise it would break for all the users.

Reply:
done